### PR TITLE
Change default lr configuration

### LINKED ISSUE #426: Investigate effects of learning rate, learning rate schedules
I've been observing that for models that take a large amount of steps to reach the early stopping criteria (~20k+ steps), increasing the learning rate significantly (5e-5 --> 2e-4) often cuts the number of steps needed in half, which in turn cuts the training time in half. For models that take less steps to begin with, an increased learning rate can also reduce the number of steps needed, but that is less often the case. The score metrics do not seem to be significantly affected by the learning rate.

To do:
* Use some hyperparameter optimization tool (ClearML, Weights and Biases?) to see if there is a learning rate that consistently reduces the training time for Scripture projects using NLLB
* Experiment with different learning rate schedules

### PR DESCRIPTION
**Pull request by isaac091**
Will update the wiki with info on using the previous lr setup

Closes #426

- - -
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/silnlp/572)

### CODE REVIEW

**Review by ddaspit — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/silnlp/572#-:-O9u0MM1D4w3XKtSGHLP:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @isaac091)
